packages/core/src/codewhisperer/service/transformByQ/transformationHubViewProvider.ts

```diff
@@ -244,29 +504,96 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
                     }
                     return timeString
                 }
+
+                function clearActiveSteps(){
+                    const activeSteps = document.querySelectorAll(".active")
+                    for(const step of activeSteps){
+                        step.classList.remove("active")
+                    }
+                }
+
+                function showStepDetails(item) {
+                    console.log("Clicked" + item.id)
```

Comment:
Nit: Remove console.log statement

Reply:
Yep, doing a pass atm. Will be removed in next push!